pkg/sfu/receiver.go

```diff
@@ -345,12 +345,11 @@ func (w *WebRTCReceiver) writeRTP(layer int) {
 		w.locks[layer].Lock()
 
 		if w.isSimulcast && len(w.pendingTracks[layer]) > 0 {
-			if pkt.KeyFrame {
-				w.downTracks[layer] = append(w.downTracks[layer], w.pendingTracks[layer]...)
-				w.pendingTracks[layer] = w.pendingTracks[layer][:0]
-			} else {
+			if !pkt.KeyFrame {
 				w.SendRTCP(pli)
 			}
+			w.downTracks[layer] = append(w.downTracks[layer], w.pendingTracks[layer]...)
+			w.pendingTracks[layer] = w.pendingTracks[layer][:0]
```

Comment:
We should fix the h264 keyframe detection, this would affect VP8 since it will change layers as soon as requested.